app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme.NoActionBar">
-        <activity android:name=".ui.MainActivity">
+        <activity
+            android:name=".ui.MainActivity"
+            android:windowSoftInputMode="stateVisible|adjustPan">
```

Comment:
Calling stateVisible here occurs only once, and upon entry into the activity. Because our app is single activity, it happens as soon as you open the app and seemed unnecessary.

I removed it in commit 1dd5c2bfbe452f47f30f59bbf5f35572b6a6c9f3.

Would like to note that the adjustPan is still not functioning properly when the keyboard is opened. Will submit an issue ticket after merge and debug her Friday.